web-interface/controlling-vms-modules/restart-vm-web.adoc

```diff
@@ -0,0 +1,16 @@
+[[restart-vm-web]]
+== Restarting a Virtual Machine
+
+A running virtual machine can be xref:vm-actions-web[restarted] from the &#8942; button found on each virtual machine in the *Workloads* -> *Virtual Machines* list, or from the xref:vm-actions-web[*Actions*] button on the *Virtual Machine Details* screen.
```

Comment:
Can we have an utf8 arrow instead of "->"?

Reply:
It looks like '->' is automatically converted into a UTF-8 arrow when the docs are created. I'm happy to change it if you think something else would look better, though.